### PR TITLE
adds cloudflare cf object to platform property

### PR DESCRIPTION
**Pull request by philippviereck**
this has been done for the pages package but not for workers. see #9978

```javascript
// @ts-expect-error req is actually a Cloudflare request not a standard request
cf: req.cf
```
This is a bit ugly, but changing the JSDoc req to `{import('@cloudflare/workers-types').Request} req` causes even more errors

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [x] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [x] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. Changesets that add features should be `minor` and those that fix bugs should be `patch`. Please prefix changeset messages with `feat:`, `fix:`, or `chore:`.

### CODE REVIEW

**Review by benmccann — APPROVED** [review]

**Comment by benmccann** [comment]
I went ahead and merged this, but I wonder if there's a reason you don't just switch to `adapter-cloudflare`? It has about 50x as much usage, so we might deprecate `@sveltejs/adapter-cloudflare-workers` at some point

https://npmcharts.com/compare/@sveltejs/adapter-cloudflare,@sveltejs/adapter-cloudflare-workers?log=true&interval=7

**Comment by philippviereck** [comment]
@benmccann there's a couple off reasons to use the workers adapter over the pages one:
- wrangler.toml
- durable objects (you need to deploy them separately when using pages) 
- control over output (the workers adapter output is the exact code that runs on CF, where as the pages adapter outputs _worker.js which gets magically converted by cloudflare to pages `Functions` -
this is an issue when working with sourcemaps)

TL;DR `@sveltejs/adapter-cloudflare-workers` offers granular control over the code deployed/running on CF.

Please don't deprecate `@sveltejs/adapter-cloudflare-workers` until cloudflare brings said features to pages. (they plan to do so I think)
